CONTRIBUTING.md

```diff
@@ -27,33 +35,38 @@ cd liveask
 ```
 
 ## Running The Local Instance
-### Dependencies
-**First Terminal**
+
+> [!IMPORTANT]
+> You need to open three terminal tabs/instances and run the following commands in each of them.
+
+### Terminal 1: Dependencies
 This is required to run up all dependencies for the application
 ```
 cd backend
-make docker-compose
+just docker-compose
 ```
-### Backend
-**Second Terminal**
+
+### Terminal 2: Backend
 This will load up the backend and connect to the dependencies
 ```
 cd backend
-make run
+just run
 ```
-### Front End
-**Third Terminal**
-The to load the frontend
+
+### Terminal 3: Front End
+Then to load the frontend
 ```
 cd frontend
-make serve
+just serve
 ```
+
 ## Configuration
-To configure the application copy the default.env to local.env & edit local.env
+To configure the application copy the `default.env` to `local.env` & edit `local.env`
 ```
 cd backend/env
 cp default.env local.env
 ```
+
 ## Notes
 - When doing local development set `RELAX_CORS` to `"1"` otherwise the backend will not get requests
-- Do not commit the index.html if only the release id has changed.
+- Do not commit the `index.html` if only the release id has changed.
```

Comment:
We probably should make an issue to ignore that finally. this is such an annoyance

Reply:
I can open an issue. Just a few questions. Why does the release id change if I am not touching the file? Is it on build? If it is, can this be stored somewhere else? Who is doing it? Rust or yew? I don't think there's a straightforward way to ignore just one line in a file using git. At least not without jumping through loops (with pre-commit hooks etc.). If there's a way to avoid this entirely, that would probably be a good approach. In any case, I can open an issue.

Reply:
> can open an issue

yeah lets definitely open another issue for that.
why does it keep changing because any build of the frontend crate runs its `build.rs` which updates the index.html. 

i fail to remember why we checked `index.html` in in the first place, it is generated by the `build.rs` during compile time based off of the `index.html.hbs` template

Reply:
Makes sense. I captured this in #83